DockerfileLeanFoundation

```diff
@@ -53,193 +51,179 @@ RUN wget https://cdn.quantconnect.com/miniconda/${CONDA} && \
+
+# Install DX Analytics
+RUN pip install git+git://github.com/yhilpisch/dx.git@4c47c258acddf966be5082530764d8115831dcb3
 
 # Install Auto-KS
-RUN wget https://cdn.quantconnect.com/auto_ks/auto_ks-master-b39e8f3.zip && \
-    unzip -q auto_ks-master-b39e8f3.zip && cd auto_ks-master && \
-    python setup.py install && cd .. && rm -irf auto_ks-master
+RUN pip install git+git://github.com/cvxgrp/auto_ks.git@b39e8f39eec33d6c55a92e2a781d62a8439f32a6
 
 # Install Pyrb
-RUN wget https://cdn.quantconnect.com/pyrb/pyrb-master-d02b56a.zip && \
-    unzip -q pyrb-master-d02b56a.zip && cd pyrb-master && \
-    python setup.py install && cd .. && rm -irf pyrb-master
+RUN pip install git+git://github.com/jcrichard/pyrb.git@d02b56ae91f67b73f39e5e474fa90b0523f6638b
 
 # Install SSM
-RUN wget https://cdn.quantconnect.com/ssm/ssm-master-34b50d4.zip && \
-    unzip -q ssm-master-34b50d4.zip && cd ssm-master && \
-    python setup.py install && cd .. && rm -irf ssm-master
+RUN pip install git+git://github.com/slinderman/ssm.git@9fd66aeddf74059c934c8b214f8f7757b9bef214
+
+# Install TA-lib for python
+RUN wget http://cdn.quantconnect.com/ta-lib/ta-lib-0.4.0-src.tar.gz && \
+    tar -zxvf ta-lib-0.4.0-src.tar.gz && cd ta-lib && \
+    ./configure --prefix=/usr && make && make install && \
+    wget https://github.com/mrjbq7/ta-lib/archive/TA_Lib-0.4.18.zip && \
+    unzip -q TA_Lib-0.4.18.zip && cd ta-lib-TA_Lib-0.4.18 && \
+    python setup.py install && cd ../.. && rm -irf ta-lib
+
+# Install py-earth
+RUN wget https://github.com/scikit-learn-contrib/py-earth/archive/0.1.0.zip && \
+    unzip -q 0.1.0.zip && cd py-earth-0.1.0 && \
+    python setup.py install && cd .. && rm -irf py-earth-0.1.0
+
+# Install fastText
+RUN wget https://github.com/facebookresearch/fastText/archive/v0.9.2.zip && \
+    unzip -q v0.9.2.zip && cd fastText-0.9.2 && \
+    python setup.py install && cd .. && rm -irf fastText-0.9.2
 
 # Install Tigramite
-RUN wget https://cdn.quantconnect.com/tigramite/tigramite-master-eee4809.zip && \
-    unzip -q tigramite-master-eee4809.zip && cd tigramite-master && \
-    python setup.py install && cd .. && rm -irf tigramite-master
+RUN wget https://github.com/jakobrunge/tigramite/archive/4.1.zip && \
+    unzip -q 4.1.zip && cd tigramite-4.1 && \
+    python setup.py install && cd .. && rm -irf tigramite-4.1
 
-# Install H2O
-RUN wget https://cdn.quantconnect.com/h2o/h2o-3.30.0.1.zip && \
-    unzip -q h2o-3.30.0.1.zip && \
-    pip install h2o-3.30.0.1/python/h2o-3.30.0.1-py2.py3-none-any.whl && \
-    rm -irf h2o-3.30.0.1
+# Install H2O: https://www.h2o.ai/download/
+RUN wget https://cdn.quantconnect.com/h2o/h2o-3.30.0.3.zip && \
+    unzip -q h2o-3.30.0.3.zip && \
+    pip install h2o-3.30.0.3/python/h2o-3.30.0.3-py2.py3-none-any.whl && \
+    rm -irf h2o-3.30.0.3
 
 # Delete temporary zip files
 RUN rm -irf *.zip *.gz
 
 # Remove black-listed packages
 RUN pip uninstall -y s3transfer
-RUN conda remove --force-remove -y s3transfer
```

Comment:
This isn't required anymore?

Reply:
Conda used to install it, but now since it's a dependency installed by pip, conda remove will throw if we call it for s3transfer